### DISCUSSION TITLE
Syncthing RC´s

### DISCUSSION THREAD

**Discussion by Kaelber** [MobiusSync/MobiusSync#15] [Q&A]
I dont have a iPhone but have a basically question.

I have switched all Synology NAS, all Windows computers and Android smartphones to v1.12.1-rc.1. This version has a special feature, as far as I can see, it contains a bug fix regarding stagnation in the synchronization.

In such cases is possible to have RC´s?

**Comment by SimonPickup** [comment]
Hi Kaelber, I'm not sure why you're asking if you don't have an iPhone.

Our expectation is to update from syncthing sources whenever there is a new full release, not on RCs.

I suppose if there is a particular need, we could do a TestFlight beta version, but hadn't expected to do this routinely.
